src/shogun/lib/GPUMatrix.h

```diff
@@ -41,6 +41,7 @@
 
 #include <shogun/lib/common.h>
 #include <memory>
+#include <viennacl/linalg/matrix_operations.hpp>
```

Comment:
This was in `Apply.h` and it is required for GPU matrix element access.
NOTE: current new GPUMatrix/Vector doesn't support element access. Maybe we can add it back.